server/autotest_server/testers/r/lib/r_renv_setup.R

```diff
@@ -0,0 +1,18 @@
+# Script to install dependencies for R tester environment using a renv.lock file. 
+
+# Install renv if not already installed
+if (!("renv" %in% installed.packages())) {
```

Comment:
This isn't the most robust way of checking package installation: see https://github.com/MarkUsProject/markus-autotesting/pull/535